src/sqlfluff/rules/L009.py

```diff
@@ -12,9 +12,10 @@
 class Rule_L009(BaseRule):
     """Files must end with a single trailing newline.
 
-    | **Anti-pattern**
-    | The content in file does not end with a single trailing newline, the $ represents
-    | end of file.
+    **Anti-pattern**
+
+    The content in file does not end with a single trailing newline, the ``$``
+    represents end of file.
```

Comment:
```suggestion
    The file does not end with a single trailing newline. (The ``$``
    represents end of file.)
```

Reply:
I hate sentences in brackets. Think it's an American thing (us Europeans would take it on to previous sentence and have full stop at end - like this).

Reply:
```suggestion
    The content in file does not end with a single trailing newline. The ``$``
    represents end of file.
```